lib/server/server.js

```diff
@@ -32,8 +33,8 @@ Meteor.methods({
     var doc;
 
     // lookup app by id if one was included
-    if (options.id) {
-      doc = Push.appCollection.findOne({ _id: options.id });
+    if (options.id || options.userId) {
+      doc = Push.appCollection.findOne({$or:[ {_id: options.id},{userId:options.userId}]});
```

Comment:
for performance it might be better to do something like:
```js
if (options.id) {
  doc = Push.appCollection.findOne({_id: options.id});
} else if (options.userId) {
  doc = Push.appCollection.findOne({userId: options.userId});
}
```